packages/nc-gui/components/cell/SingleSelect.vue

```diff
@@ -276,7 +279,13 @@ const onFocus = () => {
     @click="toggleMenu"
     @keydown.enter.stop.prevent="toggleMenu"
   >
-    <div v-if="!(active || isEditable)" class="w-full">
+    <div
+      v-if="!(active || isEditable)"
+      class="w-full"
+      :class="{
+        '!px-2': isExpandedFormOpen,
```

Comment:
most of the time, you don't need `!` for `div`. please double check. (apply to all)

Reply:
fixed